cargo-near/src/commands/build/actions/reproducible_wasm/mod.rs

```diff
@@ -51,6 +51,41 @@ pub struct BuildOpts {
     #[interactive_clap(value_enum)]
     #[interactive_clap(skip_interactive_input)]
     pub color: Option<crate::types::color_preference_cli::ColorPreferenceCli>,
+    /// Specify which named variant of `reproducible-wasm` you want to use from `Cargo.toml`
+    ///
+    /// This flag will allow you to replace settings specified in default
+    /// `[package.metadata.near.reproducible_build]`
+    /// with ones defined in
+    /// `[package.metadata.near.reproducible_build.variant.<VARIANT>]`.
+    ///
+    ///
+    /// Note that all fields in `.variant.<VARIANT>` table are optional and
+    /// will override default ones only if they are defined.
+    ///
+    /// Example:
+    /// You have developed a feature in your contract that should work only for testnet.
+    /// You could add this to your `Cargo.toml` (assuming you have already defined `testnet=[]` under `[features]`):
+    /// ```toml
+    /// [package.metadata.near.reproducible_build.variant.testnet]
+    /// container_build_command = [
+    ///     "cargo",
+    ///     "near",
+    ///     "build",
+    ///     "non-reproducible-wasm",
+    ///     "--locked",
+    ///     "--features",
+    ///     "testnet",
+    /// ]
+    /// ```
+    /// This will only replace `container_build_command` of original `reproducible_build` with
+    /// one from `.variant.testnet`.
+    ///
+    /// See also [variant flag explanation](variant-feature-explanation).
+    ///
+    /// [variant-feature-explanation]: https://github.com/near/cargo-near/tree/main?tab=readme-ov-file#custom-reproducible-wasm-build-using---variant-name-flag
+    #[interactive_clap(skip_interactive_input)]
```

Comment:
oops, i accidentally deleted an `#[interactive_clap(long)]` attr, 
re-adding it back ...